[bug] missing decrator

https://github.com/eagles-project/mam4xx/blob/ae7da4cf40f9603331ae74514678f8c6bcc6d09f/src/mam4xx/wet_dep.hpp#L61

```
/lustre/orion/cli115/proj-shared/noel/wacmy/c01-jan9/externals/ekat/src/ekat/util/ekat_scaling_factor.hpp:45:16: warning: variable 'sz2' set but not used [-Wunused-but-set-variable]
   45 |     int sz1=0, sz2=0;
      |                ^
/lustre/orion/cli115/proj-shared/noel/wacmy/c01-jan9/components/eamxx/src/physics/mam/eamxx_mam_wetscav_process_interface.cpp:547:23: error: reference to __host__ function 'init_scavimptbl' in __host__ __device__ function
  547 |         mam4::wetdep::init_scavimptbl(scavimptblvol, scavimptblnum);
      |                       ^
/lustre/orion/cli115/proj-shared/noel/e3sm_scratch/frontier-scream-gpu/c01-jan9/ndec.ne256pg2_ne256pg2.F20TR-SCREAMv1.c01-jan9.5d.n0048.sk.out10min/build/craygnuamdgpu/mpich/nodebug/threads/include/HIP/Kokkos_HIP_Parallel_Team.hpp:459:5: note: called by 'exec_team<void>'
  459 |     m_functor(member);
      |     ^
/lustre/orion/cli115/proj-shared/noel/e3sm_scratch/frontier-scream-gpu/c01-jan9/ndec.ne256pg2_ne256pg2.F20TR-SCREAMv1.c01-jan9.5d.n0048.sk.out10min/build/craygnuamdgpu/mpich/nodebug/threads/include/HIP/Kokkos_HIP_Parallel_Team.hpp:484:22: note: called by 'operator()'
  484 |       this->template exec_team<work_tag>(typename Policy::member_type(
      |                      ^
/lustre/orion/cli115/proj-shared/noel/e3sm_scratch/frontier-scream-gpu/c01-jan9/ndec.ne256pg2_ne256pg2.F20TR-SCREAMv1.c01-jan9.5d.n0048.sk.out10min/build/craygnuamdgpu/mpich/nodebug/threads/include/HIP/Kokkos_HIP_KernelLaunch.hpp:65:3: note: called by 'hip_parallel_launch_constant_memory<Kokkos::Impl::ParallelFor<(lambda at /lustre/orion/cli115/proj-shared/noel/wacmy/c01-jan9/components/eamxx/src/physics/mam/eamxx_mam_wetscav_process_interface.cpp:499:15), Kokkos::TeamPolicy<Kokkos::HIP>>>'
   65 |   driver();
      |   ^
/lustre/orion/cli115/proj-shared/noel/wacmy/c01-jan9/components/../externals/mam4xx/src/mam4xx/wet_dep.hpp:61:13: note: 'init_scavimptbl' declared here
   61 | inline void init_scavimptbl(
      |             ^
1 warning and 1 error generated when compiling for gfx90a.
```

cc @ndkeen @bartgol

--- a/src/mam4xx/wet_dep.hpp
+++ b/src/mam4xx/wet_dep.hpp
@@ -58,7 +58,8 @@ Real local_precip_production(const Real pdel, const Real source_term,
 // Function to call to initialize the arrays passe to the
 // aero_model_wetdep function. This data is constant so should
 // be allocaed on host and copied to device.
-inline void init_scavimptbl(
+KOKKOS_INLINE_FUNCTION
+void init_scavimptbl(
     Real scavimptblvol[aero_model::nimptblgrow_total][AeroConfig::num_modes()],
     Real scavimptblnum[aero_model::nimptblgrow_total]
                       [AeroConfig::num_modes()]) {

